.changeset/grumpy-worms-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@monite/sdk-react': major
+---
```

Comment:
Check [how to write a changeset description](https://www.notion.so/monite/Messages-for-Commits-MRs-Changelogs-and-Tooling-26ef378fd10141ca911ffdd4cfa6c45d?pvs=4#e7faa20bb5cf45029c9abb8ed6a83d22).